.docker/database.dockerfile

```diff
@@ -0,0 +1,20 @@
+# Database container.
+#
+# @see https://hub.docker.com/r/uselagoon/mysql-8.4/tags
+# @see https://github.com/uselagoon/lagoon-images/tree/main/images/mysql
+#
+# The ARG value will be updated with a value passed from docker-compose.yml
+ARG IMAGE=uselagoon/mysql-8.4:25.2.0
```

Comment:
Updated to 8.4

Reply:
@richardgaunt 
The production cluster will be using MySQL 8.0; I believe the whole purpose of this was to ensure compatibility across all environments - local, dev, prod.